docs/resources/runbook.md

```diff
@@ -129,7 +129,7 @@ See [Runbook Steps Configuration documentation](../runbook_steps.md) for more de
   This value _must_ be provided if `repeats` is `true`. This value _must not_ be provided if `repeats` is `false`.
 * `rule` - (Optional) JSON string representing the rule configuration for the runbook step.
   For more information on the conditional logic used in `rule`, see the
-  [Runbooks - Conditional Logic](./runbooks_conditional_logic.md) documentation.
+  [Runbooks - Conditional Logic](../guides/runbooks_conditional_logic.md)) documentation.
```

Comment:
does this have an extra paren? on the end

Reply:
yep! good catch. should be fixed now